### PR TITLE
Feature/api restructure

### PR DESCRIPTION
**Pull request by wwwsurenio**
Restructured the Networking part into a separate file called API in the Networking folder.

### CODE REVIEW

**Comment by wwwsurenio** [comment]
So I am getting somewhere, or maybe that is just my illusion. 
I get the data when the movie is clicked on, using the ID, but seems like I can only print it in the console.
Now I am not sure if I am passing it into MovieDetailsViewController...

**Comment by wwwsurenio** [comment]
So I think we can close this pull request, and maybe merge and the next step will be to fix the design?
Simulator Screen Shot - iPhone 14 - 2023-06-18 at 21 38 47

**Comment by wwwsurenio** [comment]
> Simulator Screen Shot - iPhone 14 - 2023-06-18 at 21 38 47

So my only question to make those labels that go under each of the elements or in some places above, do we create seperate cells for them too? or we add them in the current cells as vertical stacks?

File: imdb/UI/View/MovieDetailsViewController.swift [hunk 1]
```diff
@@ -27,24 +39,106 @@ class MovieDetailsViewController: UIViewController {
...
+            return cell
+            
+        case .movieOverview:
+            let cell = tableView.dequeueReusableCell(withIdentifier: "MovieOverviewCell", for: indexPath) as! MovieOverviewCell
+            cell.movieOverview = movie.movieOverview
+            return cell
+        }
+        fatalError()
+        
+        //        case 1:
```
Comment (aleksandr-lavrinenko) [inline imdb/UI/View/MovieDetailsViewController.swift:120]: Убери это в следующий

Comment (aleksandr-lavrinenko) [reply imdb/UI/View/MovieDetailsViewController.swift:120]: раз. Я про закоменченный код

**Review by aleksandr-lavrinenko — APPROVED** [review]

**Pull request merged**